webpack/assets/javascripts/react_app/components/ConfigReports/Welcome.js

```diff
@@ -1,18 +1,30 @@
 import React from 'react';
+import { FormattedMessage } from 'react-intl';
 import { translate as __ } from '../../common/I18n';
 import EmptyState from '../common/EmptyState';
 import { getManualURL, getWikiURL } from '../../common/helpers';
 
 export const WelcomeConfigReports = () => {
-  const content = __(`If you wish to configure Puppet to forward its reports to Foreman, 
-  please follow <a href=${getManualURL(
-    '3.5.4PuppetReports'
-  )}>setting up reporting</a> and
-  <a href=${getWikiURL('Mail_Notifications')}>e-mail reporting</a>`);
   const description = (
     <>
       {__("You don't seem to have any reports.")}
-      <div dangerouslySetInnerHTML={{ __html: content }} />
+
+      <br />
+      <FormattedMessage
+        id="report-manual"
+        defaultMessage={__(
+          'If you wish to configure Puppet to forward its reports to Foreman, please follow {PuppetReports} and {mail}'
+        )}
+        values={{
+          PuppetReports: (
+            <a href={getManualURL('3.5.4PuppetReports')}>
+              setting up reporting
+            </a>
+          ),
+          mail: <a href={getWikiURL('Mail_Notifications')}>e-mail reporting</a>,
```

Comment:
This won't work properly, as the portion inside the tags needs to also be translated as part of the sentence.

Reply:
Thanks! translated that too